mappings/net/minecraft/item/Item.mapping

```diff
@@ -1,149 +1,182 @@
-	METHOD a getAttributeModifiers (Lain;)Lcom/google/common/collect/Multimap;
-		ARG 1 equiptmentSlot
-	METHOD a isInItemGroup (Lbas;)Z
-	METHOD a appendItemsForGroup (Lbas;Lfj;)V
-	METHOD a getRawIdByItem (Lbbw;)I
+	METHOD a getModifiers (Lain;)Lcom/google/common/collect/Multimap;
+		ARG 1 slot
+	METHOD a isIn (Lbas;)Z
+		ARG 1 group
+	METHOD a appendStacks (Lbas;Lfj;)V
+		ARG 1 group
+		ARG 2 stacks
+	METHOD a getRawId (Lbbw;)I
 		ARG 0 item
-	METHOD a onEntityDamaged (Lbcb;Lair;Lair;)Z
+	METHOD a attack (Lbcb;Lair;Lair;)Z
 		ARG 1 stack
 		ARG 2 target
-		ARG 3 livingEntity
-	METHOD a interactWithEntity (Lbcb;Lavy;Lair;Lahc;)Z
+		ARG 3 attacker
+	METHOD a useOnEntity (Lbcb;Lavy;Lair;Lahc;)Z
 		ARG 1 stack
-		ARG 2 player
-		ARG 3 target
+		ARG 2 user
+		ARG 3 entity
+		ARG 4 hand
 	METHOD a canRepair (Lbcb;Lbcb;)Z
-		ARG 1 tool
+		ARG 1 stack
 		ARG 2 ingredient
-	METHOD a onEntityTick (Lbcb;Lbhi;Laii;IZ)V
+	METHOD a tick (Lbcb;Lbhi;Laii;IZ)V
 		ARG 1 stack
 		ARG 2 world
 		ARG 3 entity
-		ARG 4 invSlot
+		ARG 4 slot
 		ARG 5 selected
-	METHOD a onItemFinishedUsing (Lbcb;Lbhi;Lair;)Lbcb;
+	METHOD a finishUsing (Lbcb;Lbhi;Lair;)Lbcb;
 		ARG 1 stack
 		ARG 2 world
-		ARG 3 livingEntity
-	METHOD a onItemStopUsing (Lbcb;Lbhi;Lair;I)V
+		ARG 3 user
+	METHOD a onStoppedUsing (Lbcb;Lbhi;Lair;I)V
 		ARG 1 stack
 		ARG 2 world
-		ARG 3 player
-	METHOD a onBlockBroken (Lbcb;Lbhi;Lbvk;Lev;Lair;)Z
+		ARG 3 user
+		ARG 4 remainingUseTicks
+	METHOD a mine (Lbcb;Lbhi;Lbvk;Lev;Lair;)Z
 		ARG 1 stack
 		ARG 2 world
 		ARG 3 state
 		ARG 4 pos
-		ARG 5 livingEntity
-	METHOD a buildTooltip (Lbcb;Lbhi;Ljava/util/List;Lbdj;)V
+		ARG 5 miner
+	METHOD a appendTooltip (Lbcb;Lbhi;Ljava/util/List;Lbdj;)V
 		ARG 1 stack
 		ARG 2 world
 		ARG 3 tooltip
-		ARG 4 options
-	METHOD a getBlockBreakingSpeed (Lbcb;Lbvk;)F
+		ARG 4 context
+	METHOD a getMiningSpeed (Lbcb;Lbvk;)F
 		ARG 1 stack
-		ARG 2 blockState
+		ARG 2 state
 	METHOD a useOnBlock (Lbdm;)Lahd;
-		ARG 1 usageContext
-	METHOD a onUsingTick (Lbhi;Lair;Lbcb;I)V
+		ARG 1 context
+	METHOD a usageTick (Lbhi;Lair;Lbcb;I)V
 		ARG 1 world
-		ARG 2 entity
+		ARG 2 user
 		ARG 3 stack
-		ARG 4 timeLeft
+		ARG 4 remainingUseTicks
 	METHOD a use (Lbhi;Lavy;Lahc;)Lahe;
 		ARG 1 world
-		ARG 2 player
+		ARG 2 user
 		ARG 3 hand
-	METHOD a getHitResult (Lbhi;Lavy;Lbgw$b;)Lcrv;
-	METHOD a getItemFromBlock (Lbmm;)Lbbw;
+	METHOD a rayTrace (Lbhi;Lavy;Lbgw$b;)Lcrv;
+		ARG 0 world
+		ARG 1 player
+		ARG 2 fluidHandling
+	METHOD a fromBlock (Lbmm;)Lbbw;
 		ARG 0 block
-	METHOD a beforeBlockBreak (Lbvk;Lbhi;Lev;Lavy;)Z
-		ARG 1 blockState
+	METHOD a canMine (Lbvk;Lbhi;Lev;Lavy;)Z
+		ARG 1 state
 		ARG 2 world
-		ARG 3 position
-		ARG 4 player
-	METHOD a onTagDeserialized (Lib;)Z
+		ARG 3 pos
+		ARG 4 miner
+	METHOD a postProcessTag (Lib;)Z
 		ARG 1 tag
-	METHOD a getProperty (Lqs;)Lbca;
-	METHOD a addProperty (Lqs;Lbca;)V
+	METHOD a getPropertyGetter (Lqs;)Lbca;
 		ARG 1 id
-	METHOD a matches (Lzb;)Z
```

Comment:
Should `Block.matches` and `Fluid.matches` be also changed?

Reply:
Yes, however, it would be out of scope for this pull request; I intend to cover blocks and fluids separately, to avoid a huge mess of changes in one ticket.